code/app/src/main/java/com/cmput301w21t06/crowdfly/Views/ViewTrialLogActivity.java

```diff
@@ -471,6 +473,20 @@ public void onDoneGetExperiment(Experiment experiment) {
     @Override
     protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == 0) {
+            if (resultCode == RESULT_OK) {
+                HashMap<String, Object> result = (HashMap<String, Object>) data.getSerializableExtra("trialData");
+                try {
+                    Trial trialAdd = new TrialFactory().getTrialInferType(result);
+                    currentExperiment.getTrialController().addTrialData(trialAdd, expID);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+            if (resultCode == RESULT_CANCELED) {
+               // Do nothing?
```

Comment:
super small note, but maybe if the result is it has been cancelled you can have a quick log stating it's been cancelled here

Reply:
Fixed it.